index.html

```diff
@@ -12,32 +12,165 @@
 <body>
     <header class="header">
         <nav class="nav">
-            <a href="/" class="john-doe">John Doe</a>
+            <a href="/" class="john-doe">Ademibowale</a>
             <nav class="header-bar"><i class=" fa fa-bars cursor "></i></nav>
         </nav>
     </header>
     <section class="main-section ">
         <div>
             <h1 class="h1 ">Hey There.</h1>
-            <h2 class="h2">I'm Adebowale</h2>
+            <h2 class="h2">I'm Ademibowale</h2>
             <h3 class="h3">I'm a Software Developer</h3>
```

Comment:
- [ ] I love how much work you've done on your site, but again,  Try to use the right font-weight and spaces in you tests as it is from the design. 
This is your site 
![image](https://user-images.githubusercontent.com/43877689/170241645-ec694f5d-bb02-4384-9269-2aa818b46afa.png)

this is figma design
![image](https://user-images.githubusercontent.com/43877689/170241770-ebdb99fb-3924-4719-b78b-0bce7908d572.png)